tmt/base.py

```diff
@@ -2130,6 +2130,21 @@ def lint_unknown_keys(self) -> LinterReturn:
 
         yield LinterOutcome.PASS, 'correct keys are used'
 
+    def lint_empty_env_files(self) -> LinterReturn:
+        """ P001: env files are not empty """
```

Comment:
This is adding a new check with the same `P001` identifier as above. Let's use the first free slot which seems to be `P008`.

Reply:
Updated !

Reply:
Thanks!